presto-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3FileSystem.java

```diff
@@ -430,7 +438,9 @@ public boolean rename(Path src, Path dst)
             deleteObject(keyFromPath(src) + DIRECTORY_SUFFIX);
         }
         else {
-            s3.copyObject(getBucketName(uri), keyFromPath(src), getBucketName(uri), keyFromPath(dst));
+            CopyObjectRequest copyObjectRequest = new CopyObjectRequest(getBucketName(uri), keyFromPath(src), getBucketName(uri), keyFromPath(dst))
+                    .withRequesterPays(requesterPaysEnabled);
+            s3.copyObject(copyObjectRequest);
```

Comment:
I'd inline `copyObjectRequest` variable (but still retain the newline before `.withRequesterPays`)